### DISCUSSION TITLE
请问下WireGuard这种WEB管理项目是不允许上架吗

### DISCUSSION THREAD

**Discussion by dididaguai** [1Panel-dev/1Panel#6328] [General]
我在issues里面看到2个管理WireGuard的项目都被驳回了。
目前来说使用WG可以利用IPV6网络实现6to4的需求。
对于个人而言，可以在外面访问家里的服务。
对于企业来说，可以替换掉传统的ipsecvpn，sslvpn等需要较大支出的设备费用或者设备授权费用

**Comment by wanghe-fit2cloud** [comment]
感谢反馈。我们驳回部分应用上架需求主要基于以下考虑：
1. 项目不支持容器化部署。
2. 项目在 GitHub 上的 Star 数量较低（不足 1 万）。

感谢理解！
> 如果有需求，您可以将其反馈给 [社区用户维护的第三方仓库](https://github.com/okxlin/appstore)。
